From f7d05f65a1602d7da52495de0934d0d04494ea14 Mon Sep 17 00:00:00 2001
From: Jeff Backus <jeff.backus@gmail.com>
Date: Sun, 5 Jan 2014 13:19:29 -0500
Subject: [PATCH] * Modified X11_LIBS and X11_INCLUDES statements to use
 pkg-config.

---
 system-autodetect.mk | 8 +++++---
 1 file changed, 5 insertions(+), 3 deletions(-)

diff --git a/system-autodetect.mk b/system-autodetect.mk
index d61c447..69b21d0 100644
--- a/system-autodetect.mk
+++ b/system-autodetect.mk
@@ -77,12 +77,14 @@ include $(TOPDIR)/build/lua-detect.mk
 ##
 
 # Paths
-X11_PREFIX ?= /usr/X11R6
+#X11_PREFIX ?= /usr/X11R6
 # SunOS/Solaris
 #X11_PREFIX ?= /usr/openwin
 
-X11_LIBS=-L$(X11_PREFIX)/lib -lX11 -lXext
-X11_INCLUDES=-I$(X11_PREFIX)/include
+#X11_LIBS=-L$(X11_PREFIX)/lib -lX11 -lXext
+#X11_INCLUDES=-I$(X11_PREFIX)/include
+X11_LIBS := $(or $(shell pkg-config --libs x11 xext), $(error "pkg-config couldn't find X11 libraries"))
+X11_INCLUDES := $(shell pkg-config --cflags x11 xext)
 
 # XFree86 libraries up to 4.3.0 have a bug that can cause a segfault.
 # The following setting  should  work around that situation.
-- 
1.8.1.4

