diff -rupN notion-3-2013030200/contrib/styles/look_alex.lua notion-3-2013030200_patched/contrib/styles/look_alex.lua
--- notion-3-2013030200/contrib/styles/look_alex.lua	2013-03-02 16:03:45.000000000 -0500
+++ notion-3-2013030200_patched/contrib/styles/look_alex.lua	2013-11-14 05:50:46.039550572 -0500
@@ -49,7 +49,7 @@ de.defstyle("frame-floatframe", {
 
 de.defstyle("tab", {
     based_on = "*",
-    font = "-*-helvetica-medium-r-normal-*-12-*-*-*-*-*-*-*",
+    font = "-*-helvetica-medium-r-normal-*-11-*-*-*-*-*-*-*",
     de.substyle("active-selected", {
         shadow_colour = "#00aa00",
         highlight_colour = "#00aa00",
diff -rupN notion-3-2013030200/contrib/styles/look_asm.lua notion-3-2013030200_patched/contrib/styles/look_asm.lua
--- notion-3-2013030200/contrib/styles/look_asm.lua	2013-03-02 16:03:45.000000000 -0500
+++ notion-3-2013030200_patched/contrib/styles/look_asm.lua	2013-11-14 05:56:51.190094833 -0500
@@ -16,7 +16,7 @@ de.defstyle("*", {
     spacing = 0,
     -- border_style = "elevated",
     border_style = "ridge",
-    font = "-*-helvetica-medium-r-normal-*-12-*-*-*-*-*-*-*",
+    font = "-*-helvetica-medium-r-normal-*-11-*-*-*-*-*-*-*",
     text_align = "center",
 })
 
@@ -92,7 +92,7 @@ de.defstyle("tab", {
         padding_colour= "#304050",
         foreground_colour = "#a0a0a0",
     }),
-    font = "-*-helvetica-medium-r-normal-*-12-*-*-*-*-*-*-*",
+    font = "-*-helvetica-medium-r-normal-*-11-*-*-*-*-*-*-*",
     border_style = "ridge",
     padding_pixels = 0,
     shadow_pixels = 0,
@@ -122,7 +122,7 @@ de.defstyle("tab-menuentry", {
 
 de.defstyle("tab-menuentry-big", {
     based_on = "tab-menuentry",
-    font = "-*-helvetica-medium-r-normal-*-12-*-*-*-*-*-*-*",
+    font = "-*-helvetica-medium-r-normal-*-11-*-*-*-*-*-*-*",
     border_style = "inlaid",
     shadow_pixels = 0,
     highlight_pixels = 0,
diff -rupN notion-3-2013030200/contrib/styles/look_blue.lua notion-3-2013030200_patched/contrib/styles/look_blue.lua
--- notion-3-2013030200/contrib/styles/look_blue.lua	2013-03-02 16:03:45.000000000 -0500
+++ notion-3-2013030200_patched/contrib/styles/look_blue.lua	2013-11-14 05:50:19.231467396 -0500
@@ -45,7 +45,7 @@ de.defstyle("frame-tiled", {
 
 de.defstyle("tab", {
     based_on = "*",
-    font = "-*-helvetica-medium-r-normal-*-12-*-*-*-*-*-*-*",
+    font = "-*-helvetica-medium-r-normal-*-11-*-*-*-*-*-*-*",
     
     de.substyle("active-selected", {
         highlight_colour = "#9999bb",
diff -rupN notion-3-2013030200/contrib/styles/look_cleanpastel.lua notion-3-2013030200_patched/contrib/styles/look_cleanpastel.lua
--- notion-3-2013030200/contrib/styles/look_cleanpastel.lua	2013-03-02 16:03:45.000000000 -0500
+++ notion-3-2013030200_patched/contrib/styles/look_cleanpastel.lua	2013-11-14 05:55:55.078008109 -0500
@@ -14,7 +14,7 @@ de.defstyle("*", {
     shadow_pixels = 1,
     spacing = 0,
     border_style = "elevated",
-    font = "-*-helvetica-medium-r-normal-*-12-*-*-*-*-*-*-*",
+    font = "-*-helvetica-medium-r-normal-*-11-*-*-*-*-*-*-*",
     text_align = "center",
 })
 
@@ -36,7 +36,7 @@ de.defstyle("frame-ionframe", {
 
 de.defstyle("tab", {
     based_on = "*",
-    font = "-*-helvetica-bold-r-normal-*-10-*-*-*-*-*-*-*",
+    font = "-*-helvetica-bold-r-normal-*-11-*-*-*-*-*-*-*",
     de.substyle("active-selected", {
         shadow_colour = "#e0f0e0",
         highlight_colour = "#b0c0b6",
@@ -75,7 +75,7 @@ de.defstyle("tab-menuentry", {
 
 de.defstyle("tab-menuentry-big", {
     based_on = "tab-menuentry",
-    font = "-*-helvetica-medium-r-normal-*-18-*-*-*-*-*-*-*",
+    font = "-*-helvetica-medium-r-normal-*-17-*-*-*-*-*-*-*",
     padding_pixels = 10,
 })
 
diff -rupN notion-3-2013030200/contrib/styles/look_cleansteel.lua notion-3-2013030200_patched/contrib/styles/look_cleansteel.lua
--- notion-3-2013030200/contrib/styles/look_cleansteel.lua	2013-03-02 16:03:45.000000000 -0500
+++ notion-3-2013030200_patched/contrib/styles/look_cleansteel.lua	2013-11-14 05:50:03.511005660 -0500
@@ -50,7 +50,7 @@ de.defstyle("frame-tiled", {
 
 de.defstyle("tab", {
     based_on = "*",
-    font = "-*-helvetica-medium-r-normal-*-10-*-*-*-*-*-*-*",
+    font = "-*-helvetica-medium-r-normal-*-11-*-*-*-*-*-*-*",
     de.substyle("active-selected", {
         shadow_colour = "#304050",
         highlight_colour = "#708090",
diff -rupN notion-3-2013030200/contrib/styles/look_cleansteel_trans.lua notion-3-2013030200_patched/contrib/styles/look_cleansteel_trans.lua
--- notion-3-2013030200/contrib/styles/look_cleansteel_trans.lua	2013-03-02 16:03:45.000000000 -0500
+++ notion-3-2013030200_patched/contrib/styles/look_cleansteel_trans.lua	2013-11-14 05:55:03.598763445 -0500
@@ -16,7 +16,7 @@ de.defstyle("*", {
     highlight_pixels = 1,
     shadow_pixels = 1,
     border_style = "elevated",
-    font = "-*-helvetica-medium-r-normal-*-10-*-*-*-*-*-*-*",
+    font = "-*-helvetica-medium-r-normal-*-11-*-*-*-*-*-*-*",
     text_align = "center",
     transparent_background = false,
 })
@@ -55,7 +55,7 @@ de.defstyle("frame-tiled", {
 
 de.defstyle("tab", {
     based_on = "*",
-    font = "-*-helvetica-medium-r-normal-*-10-*-*-*-*-*-*-*",
+    font = "-*-helvetica-medium-r-normal-*-11-*-*-*-*-*-*-*",
     de.substyle("active-selected", {
         shadow_colour = "#304050",
         highlight_colour = "#708090",
diff -rupN notion-3-2013030200/contrib/styles/look_minimalist.lua notion-3-2013030200_patched/contrib/styles/look_minimalist.lua
--- notion-3-2013030200/contrib/styles/look_minimalist.lua	2013-03-02 16:03:45.000000000 -0500
+++ notion-3-2013030200_patched/contrib/styles/look_minimalist.lua	2013-11-14 05:53:42.695522231 -0500
@@ -20,7 +20,7 @@ de.defstyle("*", {
     highlight_pixels = 1,
     shadow_pixels = 1,
     border_style = "elevated",
-    font = "-*-helvetica-medium-r-normal-*-10-*-*-*-*-*-*-*",
+    font = "-*-helvetica-medium-r-normal-*-11-*-*-*-*-*-*-*",
     text_align = "center",
 })
 
@@ -57,7 +57,7 @@ de.defstyle("frame-floatframe", {
 
 de.defstyle("tab", {
     based_on = "*",
-    font = "-*-helvetica-medium-r-normal-*-10-*-*-*-*-*-*-*",
+    font = "-*-helvetica-medium-r-normal-*-11-*-*-*-*-*-*-*",
     padding_pixels = 0,
     highlight_pixels = 0,
     shadow_pixels = 0,
@@ -105,7 +105,7 @@ de.defstyle("tab-frame-ionframe", {
 
 de.defstyle("tab-menuentry", {
     based_on = "tab",
-    font = "-*-helvetica-medium-r-normal-*-24-*-*-*-*-*-*-*",
+    font = "-*-helvetica-medium-r-normal-*-25-*-*-*-*-*-*-*",
     text_align = "left",
     highlight_pixels = 0,
     shadow_pixels = 0,
@@ -120,7 +120,7 @@ de.defstyle("tab-menuentry", {
 
 de.defstyle("tab-menuentry-big", {
     based_on = "tab-menuentry",
-    font = "-*-helvetica-medium-r-normal-*-24-*-*-*-*-*-*-*",
+    font = "-*-helvetica-medium-r-normal-*-25-*-*-*-*-*-*-*",
     padding_pixels = 7,
     de.substyle("inactive-unselected", {
         shadow_colour = "#dfb700",
@@ -139,7 +139,7 @@ de.defstyle("input", {
     padding_pixels = 1,
     highlight_pixels = 1,
     shadow_pixels = 1,
-    font = "-*-helvetica-medium-r-normal-*-24-*-*-*-*-*-*-*",
+    font = "-*-helvetica-medium-r-normal-*-25-*-*-*-*-*-*-*",
     border_style = "elevated",
     de.substyle("*-cursor", {
         background_colour = "#000000",
diff -rupN notion-3-2013030200/contrib/styles/look_moy.lua notion-3-2013030200_patched/contrib/styles/look_moy.lua
--- notion-3-2013030200/contrib/styles/look_moy.lua	2013-03-02 16:03:45.000000000 -0500
+++ notion-3-2013030200_patched/contrib/styles/look_moy.lua	2013-11-14 05:45:03.855794018 -0500
@@ -61,7 +61,7 @@ de.defstyle("frame-floatframe", {
 
 de.defstyle("tab", {
                    based_on = "*",
-                   font = "-*-helvetica-medium-r-normal-*-12-*-*-*-*-*-*-*",
+                   font = "-*-helvetica-medium-r-normal-*-11-*-*-*-*-*-*-*",
                    de.substyle("active-selected", {
                                   shadow_colour = "#8888aa",
                                   highlight_colour = "#333366",
@@ -147,7 +147,7 @@ de.defstyle("tab-menuentry-big", {
 
 de.defstyle("input", {
                    based_on = "*",
-                   font = "-Adobe-Courier-Medium-R-Normal--12-120-75-75-M-70-ISO8859-1",
+                   font = "-Adobe-Courier-Medium-R-Normal--11-*-*-*-M-*-ISO8859-1",
                    shadow_colour = "#777777",
                    highlight_colour = "#eeeeee",
                    background_colour = "#aaaaaa",
diff -rupN notion-3-2013030200/contrib/styles/look_tibi.lua notion-3-2013030200_patched/contrib/styles/look_tibi.lua
--- notion-3-2013030200/contrib/styles/look_tibi.lua	2013-03-02 16:03:45.000000000 -0500
+++ notion-3-2013030200_patched/contrib/styles/look_tibi.lua	2013-11-14 05:51:54.258221077 -0500
@@ -106,7 +106,7 @@ de.defstyle("tab", {
     foreground_colour = "black",
     background_colour = my_neutral_color_dimmed,
     text_align = "center",
-    font = "-*-helvetica-bold-r-normal-*-10-*-*-*-*-*-*-*",
+    font = "-*-helvetica-bold-r-normal-*-11-*-*-*-*-*-*-*",
     de.substyle("active-selected", {
         background_colour = my_accent_color_normal,
     }),
@@ -127,7 +127,7 @@ de.defstyle("input-edln", {
     highlight_colour = my_accent_color_dark,
     shadow_pixels = 1,
     shadow_colour = my_accent_color_dark,
-    font ="-*-lucidatypewriter-medium-r-*-*-10-*-*-*-*-*-*-*",
+    font ="-*-lucidatypewriter-medium-r-*-*-11-*-*-*-*-*-*-*",
     de.substyle("*-cursor", {
         foreground_colour = my_accent_color_bright,
         background_colour = "black",
@@ -145,7 +145,7 @@ de.defstyle("stdisp", {
     highlight_pixels = 0,
     background_colour = my_neutral_color_dark,
     foreground_colour = "white",
-    font ="-*-lucida-medium-r-*-*-10-*-*-*-*-*-*-*",
+    font ="-*-lucida-medium-r-*-*-11-*-*-*-*-*-*-*",
 })
 
 de.defstyle("tab-menuentry", {
@@ -155,7 +155,7 @@ de.defstyle("tab-menuentry", {
     spacing = 0,
     shadow_pixels = 0,
     highlight_pixels = 0,
-    font = "-*-helvetica-medium-r-normal-*-10-*-*-*-*-*-*-*",
+    font = "-*-helvetica-medium-r-normal-*-11-*-*-*-*-*-*-*",
     foreground_colour = "black",
     background_colour = my_accent_color_bright,
     de.substyle("inactive-selected", {
@@ -172,7 +172,7 @@ de.defstyle("actnotify", {
     shadow_colour = my_accent_color_normal,
     background_colour = "red",
     foreground_colour = "white",
-    font = "-*-helvetica-bold-r-normal-*-10-*-*-*-*-*-*-*",
+    font = "-*-helvetica-bold-r-normal-*-11-*-*-*-*-*-*-*",
 })
 
 gr.refresh()
diff -rupN notion-3-2013030200/contrib/styles/look_tiny.lua notion-3-2013030200_patched/contrib/styles/look_tiny.lua
--- notion-3-2013030200/contrib/styles/look_tiny.lua	2013-03-02 16:03:45.000000000 -0500
+++ notion-3-2013030200_patched/contrib/styles/look_tiny.lua	2013-11-14 05:42:25.552155221 -0500
@@ -14,7 +14,7 @@ de.defstyle("*", {
     highlight_pixels = 0,
     shadow_pixels = 0,
     border_style = "ridge",
-    font = "-*-lucida-medium-r-normal-*-10-*-*-*-*-*-*-*",
+    font = "-*-lucida-medium-r-normal-*-11-*-*-*-*-*-*-*",
     text_align = "left",
     transparent_background = true,
 })
@@ -58,13 +58,13 @@ de.defstyle("tab", {
 
 de.defstyle("tab-menuentry", {
     based_on = "tabstyle",
-    font = "-*-lucida-medium-r-normal-*-12-*-*-*-*-*-*-*",
+    font = "-*-lucida-medium-r-normal-*-11-*-*-*-*-*-*-*",
     text_align = "left",
 })
 
 de.defstyle("tab-menuentry-big", {
     based_on = "tab-menuentry",
-    font = "-*-lucida-medium-r-normal-*-24-*-*-*-*-*-*-*",
+    font = "-*-lucida-medium-r-normal-*-25-*-*-*-*-*-*-*",
 })
 
 de.defstyle("input", {
diff -rupN notion-3-2013030200/contrib/styles/look_tiny_min_tabs.lua notion-3-2013030200_patched/contrib/styles/look_tiny_min_tabs.lua
--- notion-3-2013030200/contrib/styles/look_tiny_min_tabs.lua	2013-03-02 16:03:45.000000000 -0500
+++ notion-3-2013030200_patched/contrib/styles/look_tiny_min_tabs.lua	2013-11-14 05:51:37.054808217 -0500
@@ -16,7 +16,7 @@ de.defstyle("*", {
     highlight_pixels = 0,
     shadow_pixels = 0,
     border_style = "ridge",
-    font = "-*-lucida-medium-r-normal-*-10-*-*-*-*-*-*-*",
+    font = "-*-lucida-medium-r-normal-*-11-*-*-*-*-*-*-*",
     text_align = "left",
     transparent_background = true,
 })
@@ -73,13 +73,13 @@ de.defstyle("tab", {
 
 de.defstyle("tab-menuentry", {
     based_on = "tabstyle",
-    font = "-*-lucida-medium-r-normal-*-12-*-*-*-*-*-*-*",
+    font = "-*-lucida-medium-r-normal-*-11-*-*-*-*-*-*-*",
     text_align = "left",
 })
 
 de.defstyle("tab-menuentry-big", {
     based_on = "tab-menuentry",
-    font = "-*-lucida-medium-r-normal-*-24-*-*-*-*-*-*-*",
+    font = "-*-lucida-medium-r-normal-*-25-*-*-*-*-*-*-*",
 })
 
 de.defstyle("input", {
diff -rupN notion-3-2013030200/contrib/styles/look_whitecode.lua notion-3-2013030200_patched/contrib/styles/look_whitecode.lua
--- notion-3-2013030200/contrib/styles/look_whitecode.lua	2013-03-02 16:03:45.000000000 -0500
+++ notion-3-2013030200_patched/contrib/styles/look_whitecode.lua	2013-11-14 05:41:36.848803755 -0500
@@ -14,7 +14,7 @@ de.defstyle("*", {
     highlight_pixels = 0,
     shadow_pixels = 0,
     spacing = 1,
-    font = "-*-helvetica-medium-r-normal-*-12-*-*-*-*-*-*-*",
+    font = "-*-helvetica-medium-r-normal-*-11-*-*-*-*-*-*-*",
     text_align = "center",
 })
 
diff -rupN notion-3-2013030200/etc/look_brownsteel.lua notion-3-2013030200_patched/etc/look_brownsteel.lua
--- notion-3-2013030200/etc/look_brownsteel.lua	2013-03-02 16:02:30.000000000 -0500
+++ notion-3-2013030200_patched/etc/look_brownsteel.lua	2013-11-14 06:01:33.086483004 -0500
@@ -18,7 +18,7 @@ de.defstyle("*", {
 })
 
 de.defstyle("tab", {
-    font = "-*-helvetica-medium-r-normal-*-12-*-*-*-*-*-*-*",
+    font = "-*-helvetica-medium-r-normal-*-11-*-*-*-*-*-*-*",
     de.substyle("active-selected", {
         shadow_colour = "#304050",
         highlight_colour = "#708090",
diff -rupN notion-3-2013030200/etc/look_cleanios.lua notion-3-2013030200_patched/etc/look_cleanios.lua
--- notion-3-2013030200/etc/look_cleanios.lua	2013-03-02 16:02:30.000000000 -0500
+++ notion-3-2013030200_patched/etc/look_cleanios.lua	2013-11-14 05:51:24.088250868 -0500
@@ -19,7 +19,7 @@ de.defstyle("*", {
 })
 
 de.defstyle("tab", {
-    font = "-*-helvetica-bold-r-normal-*-10-*-*-*-*-*-*-*",
+    font = "-*-helvetica-bold-r-normal-*-11-*-*-*-*-*-*-*",
     de.substyle("active-selected", {
         shadow_colour = "#f0f066",
         highlight_colour = "#f0f066",
diff -rupN notion-3-2013030200/etc/look_clean.lua notion-3-2013030200_patched/etc/look_clean.lua
--- notion-3-2013030200/etc/look_clean.lua	2013-03-02 16:02:30.000000000 -0500
+++ notion-3-2013030200_patched/etc/look_clean.lua	2013-11-14 06:00:44.870127042 -0500
@@ -63,7 +63,7 @@ dopath("lookcommon_clean")
 
 de.defstyle("tab-menuentry-big", {
     padding_pixels = 7,
-    font = "-misc-fixed-medium-r-*-*-18-*-*-*-*-*-*-*",
+    font = "-misc-fixed-medium-r-*-*-17-*-*-*-*-*-*-*",
 })
 
 gr.refresh()
diff -rupN notion-3-2013030200/etc/look_cleanviolet.lua notion-3-2013030200_patched/etc/look_cleanviolet.lua
--- notion-3-2013030200/etc/look_cleanviolet.lua	2013-03-02 16:02:30.000000000 -0500
+++ notion-3-2013030200_patched/etc/look_cleanviolet.lua	2013-11-14 05:51:18.327447566 -0500
@@ -30,7 +30,7 @@ de.defstyle("*", {
 
 
 de.defstyle("tab", {
-    font = "-*-helvetica-medium-r-normal-*-12-*-*-*-*-*-*-*",
+    font = "-*-helvetica-medium-r-normal-*-11-*-*-*-*-*-*-*",
     
     de.substyle("active-selected", {
         -- Violet tab
diff -rupN notion-3-2013030200/etc/look_dusky.lua notion-3-2013030200_patched/etc/look_dusky.lua
--- notion-3-2013030200/etc/look_dusky.lua	2013-03-02 16:02:30.000000000 -0500
+++ notion-3-2013030200_patched/etc/look_dusky.lua	2013-11-14 05:59:29.718689478 -0500
@@ -18,7 +18,7 @@ de.defstyle("*", {
 })
 
 de.defstyle("tab", {
-    font = "-*-helvetica-medium-r-normal-*-12-*-*-*-*-*-*-*",
+    font = "-*-helvetica-medium-r-normal-*-11-*-*-*-*-*-*-*",
     de.substyle("active-selected", {
         shadow_colour = "#452727",
         highlight_colour = "#866868",
diff -rupN notion-3-2013030200/etc/look_greenlight.lua notion-3-2013030200_patched/etc/look_greenlight.lua
--- notion-3-2013030200/etc/look_greenlight.lua	2013-03-02 16:02:30.000000000 -0500
+++ notion-3-2013030200_patched/etc/look_greenlight.lua	2013-11-14 05:51:06.854839371 -0500
@@ -35,7 +35,7 @@ de.defstyle("frame", {
 
 
 de.defstyle("tab", {
-    font = "-*-helvetica-medium-r-normal-*-10-*-*-*-*-*-*-*",
+    font = "-*-helvetica-medium-r-normal-*-11-*-*-*-*-*-*-*",
     spacing = 1,
     
     de.substyle("active-selected", {
diff -rupN notion-3-2013030200/etc/look_greyviolet.lua notion-3-2013030200_patched/etc/look_greyviolet.lua
--- notion-3-2013030200/etc/look_greyviolet.lua	2013-03-02 16:02:30.000000000 -0500
+++ notion-3-2013030200_patched/etc/look_greyviolet.lua	2013-11-14 05:59:24.253875810 -0500
@@ -18,7 +18,7 @@ de.defstyle("*", {
 })
 
 de.defstyle("tab", {
-    font = "-*-helvetica-medium-r-normal-*-12-*-*-*-*-*-*-*",
+    font = "-*-helvetica-medium-r-normal-*-11-*-*-*-*-*-*-*",
     de.substyle("active-selected", {
         shadow_colour = "#333366",
         highlight_colour = "#aaaacc",
diff -rupN notion-3-2013030200/etc/look_ios.lua notion-3-2013030200_patched/etc/look_ios.lua
--- notion-3-2013030200/etc/look_ios.lua	2013-03-02 16:02:30.000000000 -0500
+++ notion-3-2013030200_patched/etc/look_ios.lua	2013-11-14 05:50:59.735082584 -0500
@@ -18,7 +18,7 @@ de.defstyle("*", {
 })
 
 de.defstyle("tab", {
-    font = "-*-helvetica-bold-r-normal-*-10-*-*-*-*-*-*-*",
+    font = "-*-helvetica-bold-r-normal-*-11-*-*-*-*-*-*-*",
     de.substyle("active-selected", {
         shadow_colour = "#f09000",
         highlight_colour = "#f0f066",
diff -rupN notion-3-2013030200/etc/look_newviolet.lua notion-3-2013030200_patched/etc/look_newviolet.lua
--- notion-3-2013030200/etc/look_newviolet.lua	2013-03-02 16:02:30.000000000 -0500
+++ notion-3-2013030200_patched/etc/look_newviolet.lua	2013-11-14 05:50:52.711322563 -0500
@@ -28,7 +28,7 @@ de.defstyle("*", {
 
 
 de.defstyle("tab", {
-    font = "-*-helvetica-medium-r-normal-*-12-*-*-*-*-*-*-*",
+    font = "-*-helvetica-medium-r-normal-*-11-*-*-*-*-*-*-*",
     
     de.substyle("active-selected", {
         highlight_colour = "#aaaacc",
diff -rupN notion-3-2013030200/etc/look_simpleblue.lua notion-3-2013030200_patched/etc/look_simpleblue.lua
--- notion-3-2013030200/etc/look_simpleblue.lua	2013-03-02 16:02:30.000000000 -0500
+++ notion-3-2013030200_patched/etc/look_simpleblue.lua	2013-11-14 05:56:58.301852342 -0500
@@ -35,7 +35,7 @@ de.defstyle("frame", {
 })
 
 de.defstyle("tab", {
-    font = "-*-helvetica-medium-r-normal-*-12-*-*-*-*-*-*-*",
+    font = "-*-helvetica-medium-r-normal-*-11-*-*-*-*-*-*-*",
     de.substyle("active-selected", {
         shadow_colour = "black",
         highlight_colour = "black",
